test/functional/lua/vim_spec.lua

```diff
@@ -579,19 +598,19 @@ describe('lua stdlib', function()
       return vim.tbl_islist(c) and count == 0
     ]]))
 
-    eq('Error executing lua: .../shared.lua: invalid "behavior": nil',
+    eq('.../helpers.lua:0: Error executing lua: shared.lua:0: invalid "behavior": nil',
```

Comment:
- It's kinda ugly that helpers.lua is here, but that's what is actually happening so no point in hiding it. 
- The `.../shared.lua` was wrong, there was no full path to `shared.lua`. The new form is correct.